### PR TITLE
[CfgEditor] Rename command names

### PR DESCRIPTION
**Pull request by dayo09**
This commit renames command names from
onevscode.* to one.editor.* and so on.

Related to #728

ONE-vscode-DCO-1.0-Signed-off-by: dayo09 <dayoung.lee@samsung.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,13 +93,13 @@
         "icon": "$(add)"
       },
       {
-        "command": "onevscode.flip-cfg-to-gui",
+        "command": "one.editor.open",
```
Comment (llFreetimell) [inline package.json:96]: If we introduce new type of custom editor, we cannot distinguish from it.
So I think adding `cfg` word seems better :)

```suggestion
        "command": "one.editor.openCfg",
```

Comment (dayo09) [reply package.json:96]: Do we plan to? I thought we plan to add more backend but not a cfg editor..

Comment (dayo09) [reply package.json:96]: What about renaming it later then? :-D

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**